nilearn/glm/tests/test_second_level.py

```diff
@@ -72,11 +74,10 @@ def fake_fmri_data(shape=SHAPE, file_path=None):
     return func_img, mask
 
 
-def test_non_parametric_inference_with_flm_objects():
+def test_non_parametric_inference_with_flm_objects(shape_3d_default):
```

Comment:
`shape_4d_default` could be used instead of `shape_3d_default` ?

Reply:
To be used later at line 80 `shapes=[(*shape_3d_default, 15)]`.

Reply:
will check

Reply:
getting some weird test failure on this one when I do this